src/java/com/puppetlabs/puppetserver/ProcessWrapper.java

```diff
@@ -48,7 +48,7 @@ public int getExitCode() {
             this.pw = pw;
         }
 
-        @Override
+        //@Override
```

Comment:
Ah, that was an accident, sorry about that, IntelliJ was being weird and this was part of my attempt to fix it. I'll uncomment that.